monitoring/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>monitoring</artifactId>
+  <packaging>jar</packaging>
+  <name>Microservices activity monitoring</name>
+
+  <parent>
+    <groupId>io.silverware</groupId>
+    <artifactId>silverware-parent</artifactId>
+    <version>2.1-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.8.4</version>
```

Comment:
Do not define a version here. Put `jackson-core` to the dependency management in our BOM and use the version already defined by `${version.jackson}` property.